widget_command.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"image"
+	"os/exec"
+	"strings"
+
+	"github.com/muesli/streamdeck"
+)
+
+// CommandWidget is a widget displaying the output of command(s)
+type CommandWidget struct {
+	BaseWidget
+	command string
+	font    string
+}
+
+func NewCommandWidget(bw BaseWidget, opts WidgetConfig) (*CommandWidget, error) {
+	bw.setInterval(opts.Interval, 1000)
+
+	var command, font string
+	ConfigValue(opts.Config["command"], &command)
```

Comment:
Ignoring errors when converting to a string is fine, we should just mark it as an intentional action:

```go
_ = ConfigValue(...)
```

This makes it clear to other developers (and the linter) that we're aware there's an error we don't want to handle.